lib/modules/dosomething/dosomething_campaign/dosomething_campaign.admin.inc

```diff
@@ -349,6 +349,16 @@ function dosomething_campaign_admin_config_form($form, &$form_state) {
     '#default_value' => variable_get('dosomething_campaign_win_copy', t('[campaign_progress] [node:field_reportback_noun] [node:field_reportback_verb]! That’s amazing - thanks to everyone for rocking [node:title]. Let’s keep it up! Add your contribution in the prove it section below.')),
     '#description' => t("Sets the win module copy"),
   ];
+  $form['win']['dosomething_campaign_win_background_image'] = [
+    '#type' => 'managed_file',
+    '#title' => t('Win module background image'),
+    '#default_value' => variable_get('dosomething_campaign_win_background_image'),
+    '#description' => t('Sets the win module background image'),
+    '#upload_location' => file_default_scheme() . '://win_module/',
```

Comment:
Let's just add this to the images folder in the theme directory since it isn't per campaign